iothub/service/src/HttpRegistryManager.cs

```diff
@@ -54,12 +54,12 @@ internal class HttpRegistryManager : RegistryManager
         private IHttpClientHelper _httpClientHelper;
         private readonly string _iotHubName;
 
-        internal HttpRegistryManager(IotHubConnectionString connectionString, HttpTransportSettings transportSettings)
+        internal HttpRegistryManager(IotHubConnectionProperties connectionProperties, HttpTransportSettings transportSettings)
```

Comment:
All these properties are now in a base class, making the HttpRegistryManager independent of the auth type.